ext/opentelemetry-ext-django/src/opentelemetry/ext/django/__init__.py

```diff
@@ -0,0 +1,93 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+
+from django import VERSION
+from django.conf import settings
+
+from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.configuration import Configuration
+from opentelemetry.ext.django.middleware import OpenTelemetryMiddleware
+
+_logger = getLogger(__name__)
+
+
+class DjangoInstrumentor(BaseInstrumentor):
+    """A instrumentor for flask.Django
+
+    See `BaseInstrumentor`
+    """
+
+    _opentelemetry_middleware = ".".join(
+        [
+            OpenTelemetryMiddleware.__module__,
+            OpenTelemetryMiddleware.__qualname__,
+        ]
+    )
+
+    def __init__(self):
+        super().__init__()
+        # Django Middleware is code that is executed before and/or after a
+        # request. Read about Django middleware here:
+        # https://docs.djangoproject.com/en/3.0/topics/http/middleware/
+
+        # This method must set this Django settings attribute:
```

Comment:
This comments can be removed, as these are talking about versions from ages ago.

Reply:
Removed version checking